os/fs/tmpfs/Kconfig

```diff
@@ -0,0 +1,73 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config FS_TMPFS
+	bool "TMP file system"
+	default n
+	depends on !DISABLE_MOUNTPOINT
+	select FS_READABLE
+	select FS_WRITABLE
+	---help---
+		Enable TMPFS filesystem support
```

Comment:
Some place has ```TMPFS filesystem``` and other has ```TMP file system```. What is correct word?
I saw 6 ```tmpfs file ...```

Reply:
I think tmpfs file system is the correct one. Will change it accordingly.